src/components/01-atoms/Tooltip/TooltipContent.test.js

```diff
@@ -15,7 +15,7 @@ describe('<TooltipContent />', () => {
       name: 'some guy',
     },
     isOwnedByUser: true,
-    isEditingMeeting: false,
+    userAction: '',
```

Comment:
Consider a constant that has the value '', so it has a name (like NO_ACTION, or something better)  ?

Reply:
Hmm, or maybe not, given the use of literals and regex elsewhere for actual action values.  Tricky.

Reply:
OK, one more random thought to consider and/or ignore:  I haven't fully grokked the code here, and this might be entirely too "Java-ey", and possibly impractical, but would it be possible/interesting to encapsulate the user action in a class representation (UserAction)?  Then, you can encapsulate the regex stuff under methods of the class.  That class becomes pretty easily testable, and potentially mock-able to make code that is dependent upon that new class more easily testable.

I guess the alternative would be to stick with the literal/scalar values here and have an "interpreter" class that you pass the value to in order to lock down the interpretation of the values, and again make that testable independently.

It's all about making "policy" things testable and regression-resistant.

Just some food for thought.  Sorry for the word salad.

Reply:
Was I somehow misinterpreting the diffs?  If so I'm more doddering than I worry about.  There are multiple places where you say "now it's like this", and that way is not what I was seeing on the branch (right-hand) side.  :-\